src/brands/peapods/lib/theme/components/peaDialog.js

```diff
@@ -2,6 +2,15 @@
 export default ({ spacing, breakpoints, white, palette }) => ({
   MuiDialog: {
     root: {
+      [breakpoints.only('xs')]: {
+        '& .MuiDialog-paper': {
+          borderRadius: 0,
+          maxHeight: 'unset',
+          height: '100%',
+          margin: 0,
+          animation: 'slideUp .5s',
```

Comment:
Here I used animation named `slideup`. 
I am not sure where we should define animation in mui-treasury.
In this file I tried to follow this but did not work: https://github.com/mui-org/material-ui/issues/13793
I think it is because it is theme definition but not sure exactly.

For now `slideUp` is defined here: https://github.com/peapods/peapods.com/pull/384/files#diff-fbaf3e4c0841b82e220396ba572a278aR72 ( I think it is not good )

Another point is that this animation is only applied when dialog is appeared.
When the dialog is disappeared, animation is not applied because component is unmounted.
To resolve this we need to use 3rd party lib like https://www.npmjs.com/package/framer-motion

Please let me know your thought.

Reply:
I agree. I updated this PR to use that library. Only the Event and Group forms are working properly on mobile however. I will leave it to you to refactor the EventFilters, UserFilters, and PodForm components in another PR.